INSTALL.md

```diff
@@ -141,65 +141,62 @@ packages necessary for building .debs, so be prepared to type your password for
+* ```wget https://jenkins.libtoxcore.so/job/qTox%20OS%20X/lastSuccessfulBuild/artifact/dep.zip```
+* ```unzip dep.zip```
 
-Next you need to download QtTools (http://qt-project.org/downloads), at the time of writing this is at version .3.0.
-Make sure you deselect all the unnecessary components from the 5.3 checkbox (iOS/Android libs) otherwise you will end up with a very large download.
+As everything from toxcore to filter audio to our special openal fork with echo cancelation is now installed, 
+laugh at some Linux users waiting for a package manager to catch up and figure out what things are called.
 
-Once that is installed you will most likely need to set the path for qmake. To do this, open up terminal and paste in the following;
+###Compiling
 
-```bash
-export PATH=/location/to/qmake/binary:$PATH
-```
+Either open Qt creator and hit build or run qmake && make in your qTox folder and it'll just work™
 
-For myself, the qmake binary was located in /Users/mouseym/Qt/5.3/clang_64/bin/.
+Note that if you use the CLI to build you'll need to add Qt5's bins to your path.
+```export PATH=$PATH:~/Qt/5.4/clang_64/bin/```
 
-This is not a permanent change, it will revert when you close the terminal window, to add it permanently you will need to add echo the above line to your .profile/.bash_profile.
+###Fixing things up
 
-Once this is installed, do the following;
+The bad news is that Qt sucks and breaks our linker paths so we need to fix those.
+First cd in to your qtox.app directory, if you used Qt Creator it's in ```~/build-qtox-Desktop_Qt_5_4_1_clang_64bit-Release``` most likely, otherwise it's in your qTox folder.
 
-```bash
-git clone https://github.com/tux3/qTox
-cd toxgui
-qmake
-```
+Install qTox so we can copy its libraries and shove the following in a script somewhere:
 
-Now, we need to create a symlink to /usr/local/lib/ and /usr/local/include/
 ```
-mkdir -p $HOME/qTox/libs
-sudo ln -s /usr/local/lib $HOME/qTox/libs/lib
-sudo ln -s /usr/local/include  $HOME/qTox/libs/include
-```
-####Final Steps
-
-The final step is to run 
-```bash
-make
-``` 
-in the qTox directory, or if you are using the bundled tox core installation, you can use 
-```bash
-./bootstrap.sh
-make
+~macdeployqt qtox.app
+cp -r /Applications/qtox.app qtox_old.app
+cp qtox.app/Contents/MacOS/qtox qtox_old.app/Contents/MacOS/qtox
+rm -rf qtox.app
+mv qtox_old.app qtox.app
 ```
-Assuming all went well you should now have a qTox.app file within the directory. Double click and it should open!
+* Give it a name like ~/deploy.qtox.sh
+* cd in to the folder with qtox.app
+* run ```bash ~/deploy.qtox.sh```
 
+###Running qTox
+You've got 2 choices, either click on the qTox app that suddenly exists, or do the following:
+* ``qtox.app/Contents/MacOS/qtox`` 
+* Enjoy the snazzy CLI output as your friends and family congratulate you on becoming a hacker
```

Comment:
@tux3 That's actually a joke, as end users on OS X don't compile it nor have they ever.